### DISCUSSION TITLE
Different Cell /Rectangle Boxes Sizes

### DISCUSSION THREAD

**Discussion by thoughtfuldata** [jsvine/pdfplumber#1014] [Ask for help with specific PDFs]
I am having trouble extracting data from this old "job application" style pages where the cell/rectangles are of different sizes and not lined up..  Any ideas on how to approach?   

Current idea is to just search for the specific titles and expand the rect into a larger one, but I was hoping to get simpler/better ideas.  possibly one with  `.find_table(s)` or `.extract_table(s)`

keep in mind the "Acid %" is blank and that's a value I want to extract.

The second empty table does not need to be extracted but I showed it for context

[example.pdf](https://github.com/jsvine/pdfplumber/files/12902757/example.pdf)

**Comment by jsvine** [comment]
The main issue you're going to run into is that this is a scanned, image-based PDF — not a digital-native/"true" PDF — and so it's not going to have the rich details re. graphical object positions, text characteristics, et cetera. As noted in the documentation, `pdfplumber` will generally be less useful for these types of PDFs, since there's less information to work with. 

That said, you might use `page.search(...)` to find the location of the text you care about, and then `page.crop(...)` to select the area beneath/around it, and then `cropped_selection.extract_text(...)` to grab the text there.
